frontend/udp/frontend.go

```diff
@@ -286,16 +286,17 @@ func (t *Frontend) handleRequest(r Request, w ResponseWriter) (actionName string
 		af = new(bittorrent.AddressFamily)
 		*af = req.IP.AddressFamily
 
+		var ctx context.Context
```

Comment:
I don't think so - this is the same as with `resp` on the next line. They're both gonna be set assigned to in line 291.